classes/classes/BodyParts/BodyPart.as

```diff
@@ -0,0 +1,47 @@
+/**
+ * Created by aimozg on 25.04.2017.
+ */
+package classes.BodyParts {
+import classes.CoC_Settings;
+import classes.Creature;
+import classes.internals.SimpleJsonable;
+
+public class BodyPart extends SimpleJsonable {
+	include "../../../includes/appearanceDefs.as";
```

Comment:
@brrritssocold No need to create function for every enum value. Leave checks against one specific value as `ContainerClass.CONSTANT_NAME` and encapsulate groups like "plain or goo or stone".